### ISSUE TITLE
Extended Crafting recipes not showing

### ISSUE THREAD

**Issue by Sereath** [friendlyhj/ZenRecipeReloading#9]
For Extended Crafting: Nomifactory Edition

The recipes do not appear until I manually reload all the recipes every time I open Minecraft again. After that, all the recipes work fine, it's just that they're not there when I join the world for the first time each relaunch.

My script with EC recipes:
https://gist.github.com/Sereath/6593e93def1319d05b52fa7fdea9a67c

This is what I added on top of it for the reloading to work as shown above:
#loader extendedcrafting reloadable
And it worked, but after that on every new launch recipes would remain hidden until a manual reload happened.

For a different script with just craftingtable recipes I have this:
#loader crafttweaker reloadable

And everything works fine.

Log for when recipes are hidden:
https://gist.github.com/Sereath/06001492d8243e39e5a22ef924258aa2

Log for when I deleted the #loader line in the script, recipes no longer hidden:
https://gist.github.com/Sereath/6fd8920767b430f69c79a049070b400e

Edit:
Likely that I'm misusing the loaders, sorry about that in advance.

**Comment by friendlyhj** [comment]
You confused `#loader` and `#modloaded` preprocessor. https://docs.blamejared.com/1.12/en/AdvancedFunctions/Preprocessors/LoaderPreprocessor
https://docs.blamejared.com/1.12/en/AdvancedFunctions/Preprocessors/ModLoadedPreprocessor

tl;dr: don't use `#loader extendedcrafting reloadable`
```zenscript
#reloadable
#modloaded extendedcrafting

// Your code about extendedcrafting recipe modification
```

**Comment by Sereath** [comment]
Thank you, I was mislead by the plugin I use showing #loader as just a comment.
